test/communication/command_line/command_line_formatter_test.exs

```diff
@@ -22,5 +22,42 @@ defmodule CommandLineFormatterTest do
 
       """
     end
+
+    test "converts an ending board into a CommandLine display friendly format" do
```

Comment:
I wonder if we need the first test at all. I think this test here is suffice to show that `format_board` can read empty slots as well as occupied ones. What do you think?

Reply:
Although I think this test is sufficient in showing the board can read empty and occupied slots, I put this starting test in there to make sure a unit test fails if you copy and paste a specific board (ie the board from the second test) into the `format_board` directly. Eli and I talked about in testing how its good to have a unit test fail when an input is "faked" ie forced to be what is in the test, so I put in two tests to make sure this happened. I hope I explained this well! If you still think it would be better to not have it I can remove it 😄, thank you for the help!

Reply:
👍 I also chatted with Eli about that so it's all good.